doc/source/code-examples/applications-by-algorithm.rst

```diff
@@ -76,4 +76,16 @@ Diagonalization Algorithms
 .. toctree::
     :maxdepth: 1
 
-    tutorials/dbi/dbi.ipynb
+    tutorials/dbi/README.md
+
+    .. tutorials/dbi/dbi_tutorial_basic_intro.ipynb
+
+    tutorials/dbi/dbi_scheduling.ipynb
+    tutorials/dbi/dbi_cost_functions_and_d_gradients_tutorial.ipynb
+    tutorials/dbi/dbi_gradient_descent_strategies.ipynb
+    tutorials/dbi/dbi_group_commutator_tests.ipynb
+
+    .. tutorials/dbi/dbi_gdbi_strategies_compare.ipynb
+
+    tutorials/dbi/dbi_strategy_Ising_model.ipynb
+    tutorials/dbi/dbi_strategy_Pauli-Z.ipynb
```

Comment:
Here is the list of notebooks currently imported in the documentation.
Feel free to re-organize them if some of them are not needed